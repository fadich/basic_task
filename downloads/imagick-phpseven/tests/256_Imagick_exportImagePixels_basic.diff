001+ pixel type 1 has incorrect values. They should be 0/255, 1/255, 2/255....or the scaled equivalent
001- Ok
002+ array(10) {
003+   [0]=>
004+   int(0)
005+   [1]=>
006+   int(13)
007+   [2]=>
008+   int(22)
009+   [3]=>
010+   int(28)
011+   [4]=>
012+   int(34)
013+   [5]=>
014+   int(38)
015+   [6]=>
016+   int(42)
017+   [7]=>
018+   int(46)
019+   [8]=>
020+   int(50)
021+   [9]=>
022+   int(53)
023+ }
024+ pixel type 2 has incorrect values. They should be 0/255, 1/255, 2/255....or the scaled equivalent
025+ array(10) {
026+   [0]=>
027+   float(0)
028+   [1]=>
029+   float(0.049835965514611)
030+   [2]=>
031+   float(0.084946974898909)
032+   [3]=>
033+   float(0.11070420386053)
034+   [4]=>
035+   float(0.13180743114366)
036+   [5]=>
037+   float(0.15001144426642)
038+   [6]=>
039+   float(0.16618600747692)
040+   [7]=>
041+   float(0.18086518654154)
042+   [8]=>
043+   float(0.19435416189822)
044+   [9]=>
045+   float(0.20689707789731)
046+ }
047+ pixel type 3 has incorrect values. They should be 0/255, 1/255, 2/255....or the scaled equivalent
048+ array(10) {
049+   [0]=>
050+   float(0)
051+   [1]=>
052+   float(0.049835965037346)
053+   [2]=>
054+   float(0.084946975111961)
055+   [3]=>
056+   float(0.11070420593023)
057+   [4]=>
058+   float(0.13180743157864)
059+   [5]=>
060+   float(0.15001145005226)
061+   [6]=>
062+   float(0.16618600487709)
063+   [7]=>
064+   float(0.18086518347263)
065+   [8]=>
066+   float(0.19435416162014)
067+   [9]=>
068+   float(0.20689707994461)
069+ }
070+ pixel type 5 has incorrect values. They should be 0/255, 1/255, 2/255....or the scaled equivalent
071+ array(10) {
072+   [0]=>
073+   int(0)
074+   [1]=>
075+   int(214043842)
076+   [2]=>
077+   int(364844479)
078+   [3]=>
079+   int(475470935)
080+   [4]=>
081+   int(566108606)
082+   [5]=>
083+   int(644294247)
084+   [6]=>
085+   int(713763467)
086+   [7]=>
087+   int(776810061)
088+   [8]=>
089+   int(834744769)
090+   [9]=>
091+   int(888616183)
092+ }
093+ pixel type 6 has incorrect values. They should be 0/255, 1/255, 2/255....or the scaled equivalent
094+ array(10) {
095+   [0]=>
096+   int(0)
097+   [1]=>
098+   int(3266)
099+   [2]=>
100+   int(5567)
101+   [3]=>
102+   int(7255)
103+   [4]=>
104+   int(8638)
105+   [5]=>
106+   int(9831)
107+   [6]=>
108+   int(10891)
109+   [7]=>
110+   int(11853)
111+   [8]=>
112+   int(12737)
113+   [9]=>
114+   int(13559)
115+ }
116+ pixel type 7 has incorrect values. They should be 0/255, 1/255, 2/255....or the scaled equivalent
117+ array(10) {
118+   [0]=>
119+   int(0)
120+   [1]=>
121+   int(3266)
122+   [2]=>
123+   int(5567)
124+   [3]=>
125+   int(7255)
126+   [4]=>
127+   int(8638)
128+   [5]=>
129+   int(9831)
130+   [6]=>
131+   int(10891)
132+   [7]=>
133+   int(11853)
134+   [8]=>
135+   int(12737)
136+   [9]=>
137+   int(13559)
138+ }
139+ pixel type 4 has incorrect values. They should be 0/255, 1/255, 2/255....or the scaled equivalent
140+ array(10) {
141+   [0]=>
142+   int(0)
143+   [1]=>
144+   int(214043842)
145+   [2]=>
146+   int(364844479)
147+   [3]=>
148+   int(475470935)
149+   [4]=>
150+   int(566108606)
151+   [5]=>
152+   int(644294247)
153+   [6]=>
154+   int(713763467)
155+   [7]=>
156+   int(776810061)
157+   [8]=>
158+   int(834744769)
159+   [9]=>
160+   int(888616183)
161+ }
162+ Ok