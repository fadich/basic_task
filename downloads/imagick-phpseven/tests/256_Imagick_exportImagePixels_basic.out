pixel type 1 has incorrect values. They should be 0/255, 1/255, 2/255....or the scaled equivalent
array(10) {
  [0]=>
  int(0)
  [1]=>
  int(13)
  [2]=>
  int(22)
  [3]=>
  int(28)
  [4]=>
  int(34)
  [5]=>
  int(38)
  [6]=>
  int(42)
  [7]=>
  int(46)
  [8]=>
  int(50)
  [9]=>
  int(53)
}
pixel type 2 has incorrect values. They should be 0/255, 1/255, 2/255....or the scaled equivalent
array(10) {
  [0]=>
  float(0)
  [1]=>
  float(0.049835965514611)
  [2]=>
  float(0.084946974898909)
  [3]=>
  float(0.11070420386053)
  [4]=>
  float(0.13180743114366)
  [5]=>
  float(0.15001144426642)
  [6]=>
  float(0.16618600747692)
  [7]=>
  float(0.18086518654154)
  [8]=>
  float(0.19435416189822)
  [9]=>
  float(0.20689707789731)
}
pixel type 3 has incorrect values. They should be 0/255, 1/255, 2/255....or the scaled equivalent
array(10) {
  [0]=>
  float(0)
  [1]=>
  float(0.049835965037346)
  [2]=>
  float(0.084946975111961)
  [3]=>
  float(0.11070420593023)
  [4]=>
  float(0.13180743157864)
  [5]=>
  float(0.15001145005226)
  [6]=>
  float(0.16618600487709)
  [7]=>
  float(0.18086518347263)
  [8]=>
  float(0.19435416162014)
  [9]=>
  float(0.20689707994461)
}
pixel type 5 has incorrect values. They should be 0/255, 1/255, 2/255....or the scaled equivalent
array(10) {
  [0]=>
  int(0)
  [1]=>
  int(214043842)
  [2]=>
  int(364844479)
  [3]=>
  int(475470935)
  [4]=>
  int(566108606)
  [5]=>
  int(644294247)
  [6]=>
  int(713763467)
  [7]=>
  int(776810061)
  [8]=>
  int(834744769)
  [9]=>
  int(888616183)
}
pixel type 6 has incorrect values. They should be 0/255, 1/255, 2/255....or the scaled equivalent
array(10) {
  [0]=>
  int(0)
  [1]=>
  int(3266)
  [2]=>
  int(5567)
  [3]=>
  int(7255)
  [4]=>
  int(8638)
  [5]=>
  int(9831)
  [6]=>
  int(10891)
  [7]=>
  int(11853)
  [8]=>
  int(12737)
  [9]=>
  int(13559)
}
pixel type 7 has incorrect values. They should be 0/255, 1/255, 2/255....or the scaled equivalent
array(10) {
  [0]=>
  int(0)
  [1]=>
  int(3266)
  [2]=>
  int(5567)
  [3]=>
  int(7255)
  [4]=>
  int(8638)
  [5]=>
  int(9831)
  [6]=>
  int(10891)
  [7]=>
  int(11853)
  [8]=>
  int(12737)
  [9]=>
  int(13559)
}
pixel type 4 has incorrect values. They should be 0/255, 1/255, 2/255....or the scaled equivalent
array(10) {
  [0]=>
  int(0)
  [1]=>
  int(214043842)
  [2]=>
  int(364844479)
  [3]=>
  int(475470935)
  [4]=>
  int(566108606)
  [5]=>
  int(644294247)
  [6]=>
  int(713763467)
  [7]=>
  int(776810061)
  [8]=>
  int(834744769)
  [9]=>
  int(888616183)
}
Ok